src/MLOps.NET/Storage/Repositories/DeploymentRepository.cs

```diff
@@ -47,6 +47,11 @@ public async Task<DeploymentTarget> CreateDeploymentTargetAsync(string deploymen
                 CreatedDate = clock.UtcNow
             };
 
+            if (isProduction)
```

Comment:
I don't think we need this if statement. We can just set the `IsProduction` flag directly :)

Reply:
Haha whoops 😊 Resolved it here: https://github.com/aslotte/MLOps.NET/pull/295/commits/b9d5bb7918d768a2ad379204dfc07a34de27071d